runtime/jvmti/jvmtiStartup.c

```diff
@@ -1110,6 +1282,77 @@ isAgentLibraryLoaded(J9JavaVM *vm, const char *library, BOOLEAN decorate)
 			j9mem_free_memory(agentPath);
 		}
 		Trc_JVMTI_isAgentLibraryLoaded_result(library, agentPath, result);
+#elif defined(AIXPPC) /* defined(LINUX) || defined(OSX) || defined(WIN32) */
+		/* The incoming library & decorate could be in the following cases:
+		 * - JvmtiAgent1 & TRUE
+		 * - libJvmtiAgent1.so & FALSE
+		 * - /path/to/libJvmtiAgent1.so & FALSE
+		 */
+		struct ld_info *objectFiles = NULL;
+		UDATA objectFileSize = 150 * sizeof(struct ld_info);
```

Comment:
A comment is warranted to explain where `150` comes from.

Reply:
Added a comment.